template_feed/Microsoft.DotNet.Common.ProjectTemplates.7.0/content/ClassLibrary-CSharp/.template.config/ide.host.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "icon": "ide/ClassLibraryCS.ico",
+  "order": 20,
+  "tags": [
+    {
+      "type": "projectType",
+      "add": [],
+      "remove": [ "Common" ]
+    }
+  ],
+  "symbolInfo": [
```

Comment:
@sayedihashimi @phenning do we need to enable framework?

Reply:
If we don't want/need it in the CLI, I'd actually prefer if we'd remove it from template.json.